packages/circuit-ui/components/Checkbox/__snapshots__/Checkbox.spec.tsx.snap

```diff
@@ -1,10 +1,932 @@
+  position: absolute;
+  top: 12px;
+  left: 0;
+  -webkit-transform: translateY(-50%) scale(0, 0);
+  -moz-transform: translateY(-50%) scale(0, 0);
+  -ms-transform: translateY(-50%) scale(0, 0);
+  transform: translateY(-50%) scale(0, 0);
+  -webkit-transition: -webkit-transform 120ms ease-in-out,opacity 120ms ease-in-out;
+  transition: transform 120ms ease-in-out,opacity 120ms ease-in-out;
+}
+
+.circuit-4:hover+label::before {
+  border-color: #666;
+}
+
+.circuit-4:focus+label::before {
+  outline: 0;
+  box-shadow: 0 0 0 4px #AFD0FE;
+  border-color: #3063E9;
+}
+
+.circuit-4:focus+label::before::-moz-focus-inner {
+  border: 0;
+}
+
+.circuit-4:focus:not(:focus-visible)+label::before {
+  box-shadow: none;
+  border-color: #999;
+}
+
+.circuit-4:checked:focus:not(:focus-visible)+label::before {
+  border-color: #3063E9;
+}
+
+.circuit-4:checked+label>svg {
+  -webkit-transform: translateY(-50%) scale(1, 1);
+  -moz-transform: translateY(-50%) scale(1, 1);
+  -ms-transform: translateY(-50%) scale(1, 1);
+  transform: translateY(-50%) scale(1, 1);
+  opacity: 1;
+}
+
+.circuit-4:checked+label::before {
+  border-color: #3063E9;
+  background-color: #3063E9;
+}
+
+.circuit-5 {
+  color: #1A1A1A;
+  display: inline-block;
+  padding-left: 26px;
+  position: relative;
+  cursor: pointer;
+}
+
+<div>
+    <div>
+      <button
+        aria-labelledby="tooltip-7"
+        class="circuit-0"
+      />
+      <div
+        class="circuit-1"
+        id="tooltip-7"
+        role="tooltip"
+        style="position: absolute; top: 0px; left: 0px;"
+      >
+        This field is required.
+        <div
+          class="circuit-2"
+          style="bottom: -4px;"
+        />
+      </div>
+    </div>
+    <div
+      class="circuit-3"
+    >
+      <input
+        class="circuit-4"
+        id="checkbox_6"
+        name="name"
+        type="checkbox"
+        value=""
+      />
+      <label
+        class="circuit-5"
+        for="checkbox_6"
+      >
+        <svg
+          aria-hidden="true"
+          fill="none"
+          height="16"
+          viewBox="0 0 16 16"
+          width="16"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            d="M5.992 14.5a.992.992 0 0 1-.707-.293l-5-5A1 1 0 0 1 1.7 7.794L5.909 12l8.308-10.125a1.001 1.001 0 1 1 1.563 1.251l-9.006 11a1 1 0 0 1-.726.375l-.056-.001z"
+            fill="currentColor"
+          />
+        </svg>
+      </label>
+    </div>
+  </div>,
+  "debug": [Function],
+  "findAllByAltText": [Function],
+  "findAllByDisplayValue": [Function],
+  "findAllByLabelText": [Function],
+  "findAllByPlaceholderText": [Function],
+  "findAllByRole": [Function],
+  "findAllByTestId": [Function],
+  "findAllByText": [Function],
+  "findAllByTitle": [Function],
+  "findByAltText": [Function],
+  "findByDisplayValue": [Function],
+  "findByLabelText": [Function],
+  "findByPlaceholderText": [Function],
+  "findByRole": [Function],
+  "findByTestId": [Function],
```

Comment:
When using `render` for snapshots, you need to destructure the `container` from the return value to avoid including all of these testing methods.